src/main/java/core/mate/academy/model/TruckProducer.java

```diff
@@ -0,0 +1,14 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer {
+    @Override
+    public List<Object> get() {
+        return new ArrayList<>((Collection) new Truck());
```

Comment:
The `get()` method should return a List of `Truck` objects. You are trying to cast a single `Truck` object into a `Collection`, which will cause a `ClassCastException`. Instead, you should create a List and add `Truck` objects to it, like this: `List<Truck> trucks = new ArrayList<>(); trucks.add(new Truck()); return trucks;`.